pylc3/CMakeLists.txt

```diff
@@ -40,7 +40,12 @@ target_link_libraries(
 set_target_properties(pylc3 PROPERTIES SUFFIX .so)
 set_target_properties(pylc3 PROPERTIES PREFIX "")
 
-install(TARGETS pylc3 LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR})
+if(SKBUILD)
```

Comment:
What's SKBUILD?

Reply:
SKBUILD is a cache variable that scikit-build, the build tool used in my Python package to invoke CMake, sets when it invokes CMake (so that we can have different behavior when the module setup invokes CMake vs when we're trying to build without setuptools etc. for a test or something).